providers/standard/tests/unit/standard/sensors/test_weekday.py

```diff
@@ -128,3 +130,23 @@ def test_weekday_sensor_timeout_with_set(self):
         )
         with pytest.raises(AirflowSensorTimeout):
             op.run(start_date=WEEKDAY_DATE, end_date=WEEKDAY_DATE, ignore_ti_state=True)
+
+    @pytest.mark.skipif(not AIRFLOW_V_3_0_PLUS, reason="Skip on Airflow < 3.0")
```

Comment:
Do we need this since it has been merged with the main?

Reply:
The standard provider will still be installable on Airflow 2.10 (and possibly 2.11)